### PR TITLE
#111658494 - As a user, I don't want to see Locale

### PR DESCRIPTION
**Pull request by Mchampsee**
Remove users ability to set their own timezone.

### CODE REVIEW

File: tests/behat/features/users.feature [hunk 1]
```diff
@@ -87,6 +87,15 @@ Feature: Users can create an account.
...
     Then I should see "The changes have been saved."
 
+  Scenario: Users should not be able to set timezone
+    Given users:
+      | name | mail | field_name_first | field_name_last |
+      | genevieve | marie@peck.com | Genevieve | Peck |
+    And I am logged in as "genevieve"
+    When I visit "/user/"
+    And I click "Edit"
+    Then I should not see "Registration is not saved"
```
Comment (fluxsauce) [inline tests/behat/features/users.feature]: That's not testing the correct thing; you're looking to see if "Time zone" isn't there.

**Comment by fluxsauce** [comment]
Hi Manish, thanks for the PR! Very close, can you adjust the test to ensure that the phrase "Time zone" is missing?

**Comment by Mchampsee** [comment]
Opps..sorry about that. Not sure why I did that. Committed with the proper label.

**Comment by fluxsauce** [comment]
Looks good! I didn't have CircleCI configured to build forks, it should be on now... merging anyway, let's see if it works.

[selfie-3](https://github.com/thieman/github-selfies/)

**Comment by Mchampsee** [comment]
LOL

**Pull request merged**